.ci/openjdk17-excluded.files

```diff
@@ -492,9 +492,6 @@
 <module name="BeforeExecutionExclusionFileFilter">
   <property name="fileNamePattern" value="[\\/]test[\\/]langtools[\\/]jdk[\\/]javadoc[\\/]tool[\\/]T4994049[\\/]FileWithTabs.java$"/>
 </module>
-<module name="BeforeExecutionExclusionFileFilter">
-  <property name="fileNamePattern" value="[\\/]test[\\/]langtools[\\/]tools[\\/]javac[\\/]annotations[\\/]neg[\\/]pkg[\\/]package-info.java$"/>
-</module>
 <module name="BeforeExecutionExclusionFileFilter">
   <property name="fileNamePattern" value="[\\/]test[\\/]langtools[\\/]tools[\\/]javac[\\/]annotations[\\/]neg[\\/]AnonSubclass.java$"/>
```

Comment:
While we are at it, let's remove all duplicates. I think the person that made this file should add some check like this to their script :)

Reply:
I wonder who that person could be (e21953969fd101b1b7889bbc85814cecf209eccb), that person could add something like this-
```bash
awk -i inplace -v start='<module name="BeforeExecutionExclusionFileFilter">' -v end='</module>' '
    $0 == start { tag=$0; in_tag=1 }
    !in_tag
    in_tag && ( $0 != start && $0 != end ) { tag=tag"\n"$0 }
    $0 == end { if (!seen[tag]++) { print tag"\n"$0 }; in_tag=0 }
' openjdk17-excluded.files
```
This will remove duplicates from the final file.

Reply:
resolved.